tests/functional/pages/surveys/section_summary/property-details-summary.page.js

```diff
@@ -0,0 +1,35 @@
+// >>> WARNING THIS PAGE WAS AUTO-GENERATED - DO NOT EDIT!!! <<<
+const QuestionPage = require('../question.page');
+
+class PropertyDetailsSummaryPage extends QuestionPage {
+
+  constructor() {
+    super('property-details-summary');
+  }
+
+  insuranceTypeAnswer() { return '#insurance-type-answer-answer'; }
+
+  insuranceTypeAnswerEdit() { return '[data-qa="insurance-type-answer-edit"]'; }
```

Comment:
Why do some section summary pages have edit and some don't?

Reply:
Previously, It repeated a lot of the id's. on generation due to some code error, now that should be fixed.